helmfile/charts/networkpolicy-service/templates/cert-manager/webhook.yaml

```diff
@@ -0,0 +1,35 @@
+{{ if .Values.certManager.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: cert-manager-webhook
+  namespace: cert-manager
+spec:
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/component: webhook
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress:
+    - from:
+{{- range $IP := .Values.global.scApiserver.ips }}
+      - ipBlock:
+          cidr: {{ $IP }}
+{{- end }}
+      ports:
+        - protocol: TCP
+          port: 10250 ## admission, use api server loop
```

Comment:
I noticed some issues with the netpol I wrote for the prometheus operator webhook. So I decided to test this one as well and it does not work for me. If I apply this netpol and then try to apply a certificate I get a timeout. Example:

```
❯ kubectl apply -f cert.yaml 
Error from server (InternalError): error when creating "cert.yaml": Internal error occurred: failed calling webhook "webhook.cert-manager.io": failed to call webhook: Post "https://cert-manager-webhook.cert-manager.svc:443/mutate?timeout=10s": context deadline exceeded
```

Does this work for you in your cluster? I had to add a rather odd ip from my control-plane node to get it to work (10.233.65.0 in my case).

Reply:
is this the calico ipip ip ?

Reply:
After some discussions you can ignore this comment.
We will include the extra IPs in the lists of apiserver IPs and node IPs. That will be fixed in https://github.com/elastisys/compliantkubernetes-apps/pull/1188

Reply:
Okay!!